TTS/tts/datasets/dataset.py

```diff
@@ -42,6 +46,20 @@ def string2filename(string):
     return filename
 
 
+def get_audio_size(audiopath):
+    extension = audiopath.rpartition(".")[-1].lower()
+    if extension == "mp3":
+        audio_info = MP3(audiopath).info
```

Comment:
Is there a reason not to use `mutagen.File`?

Reply:
Indeed, it is better. I didn't found mutagen.File when I search for it